crates/rpc/rpc-api/src/reth.rs

```diff
@@ -0,0 +1,15 @@
+use jsonrpsee::{core::RpcResult, proc_macros::rpc};
+use reth_primitives::{Address, BlockId, U256};
+use std::collections::HashMap;
+
+/// Reth API namespace for reth-specific methods
```

Comment:
Question: should this doc string (or somewhere else) document the "requirements" for being included in `RethApi` and the plan for RPCs "graduating" from here?  Probably not something I should write but if we think that's desirable I can open a separate tracking issue.

Reply:
I think that is too far in the future, we can think about that when the move is needed/desired.